test/integration/dd-agent/serverspec/dd-agent_spec.rb

```diff
@@ -8,6 +8,11 @@
   it { should be_running }
 end
 
+# verifies that the agent is installed per-machine for windows
+describe windows_registry_key('HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Installer\UserData\S-1-5-18\Products\776BC1A520EDC164EBF87A0875775D9C'), :if => os[:family] == 'windows' do
```

Comment:
I was afraid of this.  I ran it twice with two kitchen converges and it worked the same way both times.  But if its none deterministic I will remove it.